test/Microsoft.OpenApi.Tests/Validations/OpenApiComponentsValidationTests.cs

```diff
@@ -26,8 +28,12 @@ public void ValidateKeyMustMatchRegularExpressionInComponents()
                 }
             };
 
+            var validator = new OpenApiValidator();
+            validator.Visit(components);
             // Act
-            bool result = components.Validate(out errors);
```

Comment:
Now, the developer should create an object of `OpenApiValidator` and check the validator.Errors? 

Why do you think the three line codes is better than the one line code call?   `components.Validate(out errors)`? 

@PerthCharern what do you think?